### PR TITLE
Upgrade to .Net Standard 2.0

### PR DESCRIPTION
**Pull request by Swoorup**
* Upgraded/removed project to .Net Standard 2.0 for the core library. 
* For tests, due to sub-dependency (MS testing framework) requirements used .Net core
* Removed `Program.fs` for tests projects.
* Also upgraded the package references and ran the tests.
* Upgraded FSharp.Core to 4.6.2

### CODE REVIEW

File: FParsec-Pipes-Test/FParsec-Pipes-Test.fsproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
```
Comment (FoggyFinder) [inline FParsec-Pipes-Test/FParsec-Pipes-Test.fsproj]: Better to use `TargetFramework` if it is a single option

Comment (Swoorup) [reply FParsec-Pipes-Test/FParsec-Pipes-Test.fsproj]: Fixed

**Comment by rspeele** [comment]
Could you explain the reason for doing this? Since we currently target .NET Standard 1.6 I would think that would provide more backwards compatibility for users who haven't yet upgraded.

**Comment by Swoorup** [comment]
This was me trying to get .net standard 2.0 support for Rezoom, little did I know it is backwards compatible. Bit useless 😄 

Not sure if you'd want to keep the internal package updates, and some minor changes.
But can be closed if not.

**Pull request closed without merging**